### ISSUE TITLE
Typescript defination file is missing options parameter

### ISSUE THREAD

**Issue by isiah-lloyd** [cozmo/jsQR#112]
When trying to use the jsQR function with Typescript configured in vscode, you'll get the warning "expected 3 parameters, got 4", if you try to change the `inversionAttempts` option. A temporary fix was to add the following to the `index.d.ts` file 

```ts
export interface Options {
  inversionAttempts?: "dontInvert" | "onlyInvert" | "attemptBoth" | "invertFirst";
}
```
and edit the function to this:
```ts
declare function jsQR(data: Uint8ClampedArray, width: number, height: number, providedOptions: Options): QRCode | null;
```

**Comment by isiah-lloyd** [comment]
It seems like the package on npm is out of date with this repo. By installing it from the git, I was able to get the updated index.d.ts

`"jsqr": "https://github.com/cozmo/jsQR",`

**Comment by cozmo** [comment]
Ah good call it seems like we never bumped the version after adding that option. Just published `v1.2.0` which should have that. Let me know if you still have this issue.
